docs/Tutorials/1_Getting_Started/example/daml/Scripts/Setup.daml

```diff
@@ -0,0 +1,114 @@
+-- - creates an account for Alice and Bob at the Bank
+-- - creates a cash instrument and credits a cash holding to Alice on her bank account
+-- - transfers the holding from Alice to Bob
+setup : Script()
+setup = do
+
+  [alice, bank, bob] <- mapA createParty $ ["Alice", "Bank", "Bob"]
+
+  -- Account Factory (it is used by the bank to create accounts).
+  accountFactoryCid <- toInterfaceContractId @Account.F <$> submit bank do
+    createCmd Account.Factory
+      with
+        provider = bank
+        observers = empty
+
+  -- Holding Factory (it is used by the bank to create holdings with the desired implementation).
+  holdingFactoryCid <- toInterfaceContractId @Holding.F <$> submit bank do
+    createCmd Fungible.Factory
+      with
+        provider = bank
+        observers = empty
+
+  -- Alice and Bob setup account @Bank
+
+  aliceRequestCid <- submit alice do
+    createCmd AccountOpenRequest
+      with
+        owner = alice
+        custodian = bank
+
+  aliceAccountCid <- submit bank do
+    exerciseCmd aliceRequestCid AccountOpenRequest_Accept
+      with
+        label = "Alice@Bank"
+        accountFactoryCid = accountFactoryCid
+        holdingFactoryCid = holdingFactoryCid
+        observers = []
+
+  bobRequestCid <- submit bob do
+    createCmd AccountOpenRequest
+      with
+        owner = bob
+        custodian = bank
+
+  bobAccount <- submit bank do
+    exerciseCmd bobRequestCid AccountOpenRequest_Accept
+      with
+        label = "Bob@Bank"
+        accountFactoryCid = accountFactoryCid
+        holdingFactoryCid = holdingFactoryCid
+        observers = [alice]
+
+  -- Bank creates the cash instrument
+  let instrumentId = Id with label = "USD"; version = "0"
+
+  cashInstrumentCid <- toInterfaceContractId @Instrument.I <$> submit bank do
+    createCmd Instrument
+      with
+        depository = bank
+        issuer = bank
+        id = instrumentId
+        observers = empty
+
+  -- Alice deposits cash at the bank
+
+  aliceRequestCid <- submit alice do
+    createCmd DepositRequest
+      with
+        account = AccountKey with owner = alice; custodian = bank; id = "Alice@Bank"
+        instrument = InstrumentKey with issuer = bank; depository = bank; id = instrumentId
+        amount = 1000.0
+
+  aliceCashHoldingCid <- submit bank do exerciseCmd aliceRequestCid CashDepositRequest_Accept
+
+  -- Bob requests a cash transfer from Alice
+
+  transferRequestCid <- submit bob do
+    createCmd TransferRequest
+      with
+        receiverAccount = AccountKey with owner = bob; custodian = bank; id = "Bob@Bank"
+        instrument = InstrumentKey with issuer = bank; depository = bank; id = instrumentId
+        amount = 1000.0
+        sender = alice
+
+  newHoldingCid <- submit alice do exerciseCmd transferRequestCid CashTransferRequest_Accept with holdingCid = aliceCashHoldingCid
+
+  pure ()
+
+createParty : Text -> Script Party
+createParty name = allocatePartyWithHint name $ PartyIdHint name
```

Comment:
add a single empty line

Reply:
Github displays something like ⛔ at the end of the last line in the file when the file doesn't end with an empty line. Also, if the file does end with have a single empty line, it doesn't get displayed in GitHub.